types/react-dev-utils/WebpackDevServerUtils.d.ts

```diff
@@ -18,17 +18,16 @@ export function choosePort(host: string, defaultPort: number): Promise<number |
 
 /**
  * Creates a Webpack compiler instance for WebpackDevServer with built-in
- * helpful messages. Takes the `require('webpack')` entry point as the first
- * argument. To provide the `urls` argument, use `prepareUrls()` described
- * below.
+ * helpful messages. Takes the `require('webpack')` entry point.
+ * To provide the `urls` argument, use `prepareUrls()` described below.
  */
-export function createCompiler(
-    _webpack: typeof webpack,
+export function createCompiler(opts: {
```

Comment:
Looks like there are some new options: https://github.com/facebook/create-react-app/tree/master/packages/react-dev-utils#createcompilerargs-object-webpackcompiler
Also would be nice to have it exported as a `CreateCompilerOptions` interface to easier usage.